lib/filetests/src/lib.rs

```diff
@@ -76,7 +76,30 @@ pub fn run(verbose: bool, files: &[String]) -> TestResult {
     }
 
     runner.start_threads();
-    runner.run()
+    runner.run(None, None)
+}
+
+/// Used for 'pass' subcommand.
+/// Commands are interpreted as test and executed.
+///
+/// Directories are scanned recursively for test cases ending in `.clif`.
+///
+pub fn run_pass(
+    verbose: bool,
+    passes: &Vec<String>,
+    target: Option<&str>,
+    file: &String,
+) -> TestResult {
+    let mut runner = TestRunner::new(verbose);
```

Comment:
Is `verbose` the right name here? The caller names it "time-passes", which had me think that it was for timing purposes. So maybe either the caller's argument or the parameter here should be fixed?